### PR TITLE
fix: cannot fullscreen due to `Permission-Policy`

### LINKED ISSUE #1942: video fullscreen button is disabled
截屏2023-04-02 09 59 23

### PR DESCRIPTION
**Pull request by alexzhang1030**
closes #1942

### CODE REVIEW

**Comment by patak-dev** [comment]
Do you see the fullscreen button enabled in the preview? I still get it disabled for a large screen https://deploy-preview-2281--elk-zone.netlify.app/universeodon.com/@CraigWeekend@mstdn.ca/110794193399419745

**Comment by alexzhang1030** [comment]
@patak-dev After a considerable amount of debugging, I've finally discovered that this issue originates from the [Permission-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy/fullscreen), and it only occurs in Chrome 😅.

Already revert the last changes. You can merge it now!

**Comment by alexzhang1030** [comment]
And why does the preview version not change? 🤔

File: nuxt.config.ts [hunk 1]
```diff
@@ -242,6 +242,9 @@ export default defineNuxtConfig({
         'style-src': ['\'self\'', '\'unsafe-inline\''],
         'upgrade-insecure-requests': true,
       },
+      permissionsPolicy: {
+        fullscreen: ['\'self\'', 'https:', 'http:'],
```
Comment (patak-dev) [inline nuxt.config.ts:246]: Does it need to be so permissive? I'll let others review this one in that case.

Comment (alexzhang1030) [reply nuxt.config.ts:246]: FYI：it's syncs with contentSecurityPolicy.media-src

Comment (patak-dev) [reply nuxt.config.ts:246]: Ah, I see.

**Comment by patak-dev** [comment]
In MacOS/Chrome, if I press the fullscreen on a video, then press exit, the app goes back to the top of the timeline. I don't know if we should merge this as it may confuse folks 🤔

**Comment by alexzhang1030** [comment]
> In MacOS/Chrome, if I press the fullscreen on a video, then press exit, the app goes back to the top of the timeline. I don't know if we should merge this as it may confuse folks 🤔

Ah, it's indeed a problem, I'll see if I can do something...

**Comment by alexzhang1030** [comment]
@patak-dev It works! Can't believe that the issue, I can't believe this is what caused the problem. Chrome sucks 🤪

**Comment by alexzhang1030** [comment]
Ah, netlify does not update the preview

**Review by patak-dev — APPROVED** [review]
Nice! Interesting that the `height` had such an effect. Let's try it out in main.elk.zone. I tested in Android and desktop and using min-height looks good.

**Pull request merged**